tests/docker_cluster.py

```diff
@@ -419,14 +434,6 @@ def write_content_to_host(self, content, path, host):
     def copy_to_host(self, source_path, dest_host):
         shutil.copy(source_path, self.get_local_mount_dir(dest_host))
 
-    @nottest
-    def clean_up_presto_test_images(self):
```

Comment:
I think it still may be necessary to clean up the test images -- that way if you're running product tests from intellij it deletes the old images.